sales.html

```diff
@@ -50,6 +50,20 @@ <h1>Hourly/Daily Sales</h1>
   <header>
 
   </header>
+
+  <form id="get-store">
+    <fieldset>
+   <p> <label for="storeName-input">Store Name</label>
+    <input type="text" id="storeName-input" name="storeName" required></input></p>
```

Comment:
Nice use of required on these inputs!

Reply:
But... still keep watching your indentation! The `p` tag should be indented 3 more spaces, and then this indented another 2 past that.